### PR TITLE
Brand cards - New pattern for cards

### PR DESCRIPTION
**Pull request by mkayan**

### CODE REVIEW

File: app/kitchensink/index.html [hunk 1]
```diff
@@ -912,7 +912,8 @@ <h2 id="Cards" class="page-header">Cards
                         <h3 class="mdl-card__title-text">CBP Super Bowl 50 Air and Marine Operations Security</h3>
                       </div>
                       <div class="mdl-card__supporting-text">
-                        Personnel with U.S. Customs and Border Protection conduct operations in advance of Super Bowl 50 in Santa Clara, Calif., 2016. (U.S. Customs and Border Protection Photo by Glenn Fawcett)
+                        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type
```
Comment (davezen1) [inline app/kitchensink/index.html]: Don't need to change this since it matches the picture and is an example

File: app/styles/custom/_utilities.scss [hunk 2]
```diff
@@ -235,6 +235,10 @@
     font-weight: 500;
 }
 
+.text-italic{
```
Comment (davezen1) [inline app/styles/custom/_utilities.scss]: why is this needed? did we not have text italic before or inherited?

File: app/styles/vendor/_material_cards.scss [hunk 3]
```diff
@@ -153,15 +159,189 @@ $card-actions-font-size: ceil(($font-size-base * 1.04)); // ~15px
...
+    /* For Safari 3.1 to 6.0 */
+    transition: width .2s, height 2s, linear .2s;
+    .image-container {
+      margin-left: auto;
+      margin-right: auto;
+      margin-top: -16px;
+      height: 125px;
+      width: 125px;
+      border-radius: 5px;
+      &.image1 {
```
Comment (davezen1) [inline app/styles/vendor/_material_cards.scss]: what's the idea behind the colors? Could we provide a more flexible way to add colors so that a user could specify?

File: app/kitchensink/kitchensink.js [hunk 4]
```diff
@@ -124,4 +124,10 @@ $(document).ready(function () {
   $('#topSecret').on('click', function () {
     hulkIt('top-secret')
   })
+
+  //brand card interaction
```
Comment (davezen1) [inline app/kitchensink/kitchensink.js]: could we provide a way to contract as well?

File: app/styles/custom/_utilities.scss (hunk 2 above)
Comment (mkayan) [reply app/styles/custom/_utilities.scss]: This style is not inherited from bootstrap, thus the new addition.

File: app/kitchensink/kitchensink.js (hunk 4 above)
Comment (mkayan) [reply app/kitchensink/kitchensink.js]: Added.

File: app/kitchensink/index.html (hunk 1 above)
Comment (mkayan) [reply app/kitchensink/index.html]: Reverted back.
